src/lib/style/abstracts/_mixins.scss

```diff
@@ -119,6 +127,9 @@ $link-underline-color-default: adjust-color(
     border-bottom: 2px solid map.get(SRC.$primary-color-palette, 700);
     margin-bottom: -1px;
   }
+  &.disabled {
+    @include disabled;
+  }
```

Comment:
Not necessary, but could consider adding `cursor: default` or `cursor: not-allowed` for disabled links (unset will set it to `cursor: text`).